baseapp/baseapp_pages/graphql/object_types.py

```diff
@@ -67,6 +68,11 @@ def resolve_target(self, info, **kwargs):
                 return None
         return self.target
 
+    @classmethod
+    def get_queryset(cls, queryset, info):
+        MAX_COMPLEXITY = 3
```

Comment:
lets move this `MAX_COMPLEXITY` to be `cls.MAX_COMPLEXITY` instead, this will make this number be easily changed per project